src/main/kotlin/antlr/treepat/Treepat.g4

```diff
@@ -1,15 +1,8 @@
-// Define a grammar called Hello
+// Define a grammar called Treepat.
```

Comment:
@jpavlich This is the updated grammar for Treepat. We are really concerned with the quality as we don't have much experience writing these type of code. Can you give use a review of the file and maybe give use some feedback regarding naming, format, readability, maintainability, etc.

We would also appreciate if you could review the `TreeFormat.g4` file. 😁

Thank you! 😊